### ISSUE TITLE
Abstract synonym definitions still produce unsolved metas

### ISSUE THREAD

**Issue by GoogleCodeExporter** [agda/agda#729]
The following gives unsolved metas at the last line. Without the `abstract` it works.

``` agda
module Test where

id : {A : Set} → A → A
id x = x

abstract
  id2 = id
```

Original issue reported on code.google.com by `guillaum...@gmail.com` on 23 Oct 2012 at 1:12

**Comment by andreasabel** [comment]
Made some progress on this issue via fix of #2102: mutual abstract aliases work now.

**Comment by UlfNorell** [comment]
Decision in #418: abstract aliases should not work
